geoips/commandline/args.py

```diff
@@ -635,6 +635,15 @@ def add_args(parser, arglist=None):
                     be passed to the algorithm in full.""",
         )
 
+    if arglist is None or "image_compare_threshold" in arglist:
+        procflow_group.add_argument(
+            "--image_compare_threshold",
+            choices=["lenient", "medium", "strict"],
+            default="medium",
+            type=str.lower,
+            help="Threshold Value for comparing Pixels in Images.",
```

Comment:
Should the help here describe what "lenient", "medium", and "strict" mean?

Reply:
Outdated. I can update the comments more in this PR, or the unit tests one, to describe exactly what threshold does for image comparisons. Ironically, it's vaguely documented in the source code for pixel match. To my understanding, threshold is the percent difference allowed to pass for image comparisons. This value is between 0 and 1; for example if compare_pix == 4, output_pix == 7, and threshold == 0.5, it would fail because 7 is greater than 50% of the compare_pix.

Reply:
@evrose54 Rather than figuring out how to document it here, can you please make note of this on the issue that discusses creating the new CLI? Basically, I'd like to point out that we need to create a set of optional CLI arguments that get dynamically brought in based on which plugin is being called and that this documentation should go there.

I don't know if that makes sense, so if it doesn't, we can talk.

Reply:
The documentation would not go in commandline/args.py, but directly in the plugins/output_checkers/image.py docstring - I think it is appropriate to put it there (since that is where the new CLI will pull the information from.

Reply:
I'm resolving this one, since description of the image plugin kwargs does not belong in commandline/args.py.  See comment:
https://github.com/NRLMMD-GEOIPS/geoips/pull/367#discussion_r1358902986

Reply:
Also added a note to the CLI issue: 
https://github.com/NRLMMD-GEOIPS/geoips/issues/37